experimental/oak_baremetal_runtime/schema.fbs

```diff
@@ -25,6 +25,15 @@ table Initialization {
     constant_response_size: uint;
 }
 
+table InitializeResponse {
+    public_key_info: PublicKeyInfo;
+}
+
+table PublicKeyInfo {
+    public_key: [ubyte];
```

Comment:
I think at some point we need to find a proper name for this key. Currently it's called public_key or signing_public_key in different places, neither of which is particularly descriptive. Perhaps root_public_key or instance_public_key or runtime_public_key or something sufficiently unambiguous would help (here, in code, and in documentation).

Reply:
The naming depends on the usage. For interactive remote attestation it will be a signing key, but for the offline attestation it will be an encryption (or DH) key. I created #3181 to track public key naming.